test/integration/query-tests/terrain/circle/map-aligned/unoccluded/expected.json

```diff
@@ -1 +1,25 @@
-[]
\ No newline at end of file
+[
```

Comment:
@arindam1993 If I'm reading this test right, there should have been a result here as the query point is right on top of an unoccluded circle?